main.go

```diff
@@ -387,6 +390,48 @@ func main() {
 
 	// Start command execution
 	cmd.HandleFunc(ctx, args)
+
+	// Go routine for checking the GPS
+	go func() {
+		if fOptions.UpdateGrid {
+			for range time.Tick(time.Minute * 5) {
+				updateGridIfNeeded()
+			}
```

Comment:
What's the reason for polling once per 5 minute? Wouldn't it be even better to continuously keep the locator field up-to-date by subscribing (Watch(true)) and call Next() in an endless loop? 🤔

Reply:
My thought was that while most people will be mostly static...i.e. once they get the first fix from the GPS, they won't need it anymore.  However, for true mobile operation, it might be required to get a new grid square at a regular interval.  

I picked 5 minutes at random for testing, but I think long term, this might be something we allow the user to enable/disable and set the interval as needed. If you were in a moving car for example, 5 minutes might be a good interval, but on a sailboat, it might need be only once every few hours.  

I'll disable the loop for now.